src/oscar/apps/dashboard/orders/views.py

```diff
@@ -356,43 +367,40 @@ def render_to_response(self, context, **response_kwargs):
     def get_download_filename(self, request):
         return 'orders.csv'
 
+    @cached_property
+    def sorted_csv_columns(self):
+        columns = OrderedDict()
```

Comment:
This method can be simplified to just:

```
return OrderedDict(self.CSV_COLUMNS)
```
which is simple enough that it may be easier just to do that where we need it, directly.